draft-irtf-cfrg-vdaf.md

```diff
@@ -1390,28 +1355,44 @@ before they can begin preparing the measurement shares for aggregation.
       V              V                      V
     out_share_0    out_share_1         out_share_[SHARES-1]
 ~~~~
-{: #prep-flow title="VDAF preparation process on the input shares for a single
-measurement. At the end of the computation, each Aggregator holds an output
-share or an error."}
+{: #prep-flow title="Illustration of interactive VDAF preparation."}
 
-To facilitate the preparation process, a concrete VDAF implements the following
-methods:
+Preparation is organized into a number of rounds. The number of rounds depends
+on the VDAF: Prio3 ({{prio3}}) has one round and Poplar1 ({{poplar1}}) has two.
+
+At the start of each round, each Aggregator broadcasts a message called a
+"preparation share", or "prep share" for short. The prep shares are then
+combined into a single message called the "preparation message", or "prep
+message". The prep message may be computed by any one of the Aggregators.
+
+The prep message is disseminated to each of the Aggregators to begin the next
+round, or compute the output shares in case of the last round. An Aggregator
+begins the first round with its input share and it begins each subsequent round
+with the previous prep message. Its output in the last round is its output
+share and its output in each of the preceding rounds is a prep share.
```

Comment:
I think these paragraphs would be a good place to introduce "prep state". Its first use is currently in the definition of `vdaf.prep_init()` below. How about this?

> Aggregators retain some local state between successive rounds of preparation. This is referred to as `preparation state` or `prep state` for short.

Reply:
Done.